docs/docs/guides/cli/how-to-use-lightdash-preview.mdx

```diff
@@ -19,4 +19,66 @@ Then `cmd` + `click` to open the preview link from your terminal. Once you're in
 
 When you create developer previews, Lightdash will use the same warehouse connection settings you have in your 
 `profiles.yml` file for your current dbt project. This can be a problem if you're using a local database that your 
-laptop can reach but your Lightdash instance cannot.
\ No newline at end of file
+laptop can reach but your Lightdash instance cannot.
+
+
+## How to automatically sync your change in Lightdash
+
+If you've connected Lightdash to GitHub, you can setup a `github action` and get Lightdash to create new dynamic `preview` projects automatically 
+when a new `pull request` is created, and it will automatically delete the `preview` project when the `pull request` is closed or merged. 
+
+## First add the credentials to Github secrets
```

Comment:
I literally copy-pasted this section from the other document, not sure what's the best way to handle this, maybe linking the other page, is there any way to `iframe` it ?

Reply:
I don't know 🤔 I've never done iframes.

Normally, I've just done `copy-paste` for similar blocks (that wouldn't be docs on their own)

Probably not the most scalable way of doing things since we'll need to update in multiple places 😅 but I think it's fine for now!